packages/compass-aggregations/src/modules/preferences-readonly.ts

```diff
@@ -0,0 +1,39 @@
+import type { AnyAction } from 'redux';
+
+/**
+ * The preferencesReadOnly changed action name.
+ */
+export const PREFERENCES_READONLY_CHANGED =
+ 'aggregations/preferences-readonly/PREFERENCES_READONLY_CHANGED';
+
+/**
+ * The initial state of the preferencesReadOnly attribute.
+ */
+export const INITIAL_STATE = false;
```

Comment:
Afaict we always initialize these as soon as the store is loaded, so is there any reason not to use the safe default of `true` here?